trace/propagation/http/tracestate/tracestate.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2018, OpenCensus Authors
```

Comment:
I am not sure if this is the right place for this package.

Wouldn't be somewhere under go.opencensus.io/plugin/ochttp/propagation is better?

We can even put it in godoc.org/go.opencensus.io/plugin/ochttp/propagation/tracecontext. It doesn't have to have its own package.

Reply:
@rakyll , so move it back to commit https://github.com/census-instrumentation/opencensus-go/pull/901/commits/442f8ebf0f41eefa1eb65ebb2c3ba916ff7391b4 ?

@reyang are you ok with moving back to ochttp?

Reply:
@rakyll, @bogdandrutu, I think we are trying to keep the header manipulation logic separate from the actual HTTP implementations, right? For instance, in Python we have Django, Flask and httplib, all are using the same tracecontext from_headers and to_headers logic.

Regarding Golang, I don't know if ochttp would be the only implementation. If we need to support another HTTP plugin, do we duplicate the tracecontext logic in ./plugin/xyzhttp/propagation/tracecontext?